Src/DependencyCollector/Shared/Implementation/HttpDiagnosticSourceListener.cs

```diff
@@ -100,6 +105,81 @@ protected virtual void Dispose(bool disposing)
                 this.disposed = true;
             }
         }
+
```

Comment:
Http reflection hook is now implemented in DiagnosticSource package, so the Src/DependencyCollector/Shared/Implementation/HttpHandlerDiagnosticListener.cs is removed.

This class implements efficient property fetching with reflection